ffs/ffs_terms.md

```diff
@@ -0,0 +1,172 @@
+
+#### (a) Idea
+
+In the Idea stage, a potential Contractor has an idea of which they think will benefit the Monero Project. To gauge community reaction, it is advisable to submit a post with the prefix [IDEA] in the title on [/r/Monero](https://www.reddit.com/r/Monero/) subreddit or bring the idea up in a Community Workgroup Meeting on IRC Freenode channel `#monero-community`. Meeting dates are published in the [Issues](https://github.com/monero-project/meta/issues) section of the GitHub Meta Repository for General Monero Project Matters. 
+
+Next, to make a formal proposal, the Contractor would register on FFS and create a new thread in the Idea section with a short title describing the idea. 
+
+The content of the of post must include the following:
+
+- **What** - description of the idea and how the Contractor plans to develop it  
+
+- **Who** - who are they are? Why they think they can develop it, and why should the Monero community believe they can?  
+
+- **Why** - Why do they want to develop this?  
+
+- **Timeline and Milestones** - It is difficult to describe how each proposal should be broken down into milestones, but the Contractor should know how long things should take and how to break tasks broken down. Each milestone should include the costs in XMR.
+
+- **Expiration** - Include time limits for the proposal. 
+
+- **Affirmation** - The Contractor must make a statement affirming that they have read these instructions and accept the terms & conditions of FFS.
+
+- **Signature** - Include a signature hash of the proposal text, [signed](https://monero.stackexchange.com/questions/4641/how-do-you-sign-a-message-with-your-monero-private-key) with Contractor’s Monero private key.
+
+A FFS Moderator may request clarifications or additional information. 
+
+Members of the community will have an opportunity to comment on the thread, voice their concerns/opposition/support, make recommendations and the Contractor may make adjustments to the proposal (and update the signature hash accordingly).
+
+Once the Contractor is satisfied with the state of the proposal, he or she can send a PM to a moderator to have the proposal moved to the Public Voting stage.
+
+#### (b) Public Voting
+
+Once in the Public Voting stage, the Contractor, or a community member, submits a link to the FFS proposal with the prefix [FFS Voting] in the title on [/r/Monero](https://www.reddit.com/r/Monero/) subreddit. Members of the community can vote to have a proposal elevated to the Funding Required stage by sending 0.01 XMR (with low priority to minimize costs) to a unique Monero address and payment ID created for the proposal by clicking on the “Vote” button on FFS. If the proposal gets at least 10 contributions, the proposal is moved up to the Funding Required stage and the funds that were raised are applied to amount being sought for the project. 
```

Comment:
>Members of the community can vote to have a proposal elevated to the Funding Required stage by sending 0.01 XMR (with low priority to minimize costs) to a unique Monero address and payment ID created for the proposal by clicking on the “Vote” button on FFS. If the proposal gets at least 10 contributions, the proposal is moved up to the Funding Required stage and the funds that were raised are applied to amount being sought for the project. 

Ouch.

1. Neglects the poor
2. Joe creates an idea that no one likes but, with 10 fake accounts, sends 10 contributions to his own proposal anyway. That proposal is then moved to Funding Required?!?

Reply:
1. The amount could be adjusted.
2. True, but to what end? Even if the proposal is moved to the Funding Required stage, the proposal still needs to meet the fully funded threshold by a deadline. If there is no support from the community, the proposal will stagnate and eventually revert back to the Idea stage. 

The alternative would be to have proposals vetted by an FFS Moderator and then manually moved up the Funding Required stage. [Decred's proposal system](https://github.com/decred/politeia) follows an off-chain solution with a censorship-resistant timestamping mechanism. Perhaps we can have something similar. I actually played arround with the idea of [timestamping data on the Monero blockchain](https://github.com/alvinjoelsantos/atesti-api) a few months ago.